### ISSUE TITLE
License

### ISSUE THREAD

**Issue by fi0** [se7en9057/Universal-Switch-Footprint#2]
Is it possible to license this under GPL and MIT? thank you!

**Comment by se7en9057** [comment]
i dont think that would be an issue but i'd like to understand the difference first.
